src/BTCPayServer.Lightning.All/LightningConnectionString.cs

```diff
@@ -377,6 +376,40 @@ private static bool TryParseNewFormat(string str, out LightningConnectionString
                     result.BaseUri = ptarmiganuri;
                     result.ApiToken = ptarmiganApiToken;
 
+                    break;
+                case LightningConnectionType.LNbank:
+                    var lnbankServer = Take(keyValues, "server");
+
+                    if (lnbankServer == null)
+                    {
+                        error = $"The key 'server' is mandatory for LNbank connection strings";
+                        return false;
+                    }
+                    if (!Uri.TryCreate(lnbankServer, UriKind.Absolute, out var lnbankUri)
+                        || (lnbankUri.Scheme != "http" && lnbankUri.Scheme != "https"))
```

Comment:
you may want to add the `allowinsecure` that we use elsewhere in case they url does not use `https` (or tor)

Reply:
Good call, will add that! Thanks.

Reply:
Done in a98aea7a5c37a0dba9724afe32eb89e960ecc566.